### ISSUE TITLE
AppCache manifests

### ISSUE THREAD

**Issue by tilgovi** [hypothesis/via#7]
We may want to strip the `manifest` attribute from the `html` tag.

I can't quite tell (and haven't tested) whether the appcache is used by pages that don't explicitly opt in if they reference resources that were cached by another page.

In other words, if Document A has a manifest attribute and lists /favicon.io then would a Document B that has no manifest but loads /favicon.ico cause Document B to load a cached version? I don't know. If so, this could be a huge problem, depending on `base` is handled with AppCache and how via handles the base URI and how paths in the manifest resolve.

**Comment by ikreymer** [comment]
Yeah, it seems that any kind of live web proxy use case should disable the AppCache. Easy enough to always strip it out at the html rewriting phase, though I suppose it might be useful in the case of an archived page, maybe?

**Comment by tilgovi** [comment]
Not sure. It might actually be totally fine. I'm just not up on the
semantics well enough to know if pages might interfere with one another. If
that's not the case, if assets are cached per page, and/or the paths in the
manifest are rooted at the base URL and that is guaranteed to be set
appropriately, then it couldn't hurt and might even help to keep it.

Otherwise, scary.
